### PR TITLE
commands>service>publish: confirm before publishing to ipfs

### PR DESCRIPTION
**Pull request by ilgooz**

### CODE REVIEW

File: commands/service_publish.go [hunk 1]
```diff
@@ -56,6 +60,18 @@ func (c *servicePublishCmd) preRunE(cmd *cobra.Command, args []string) error {
 }
 
 func (c *servicePublishCmd) runE(cmd *cobra.Command, args []string) error {
+	if !c.yes {
+		var confirmed bool
+		if err := survey.AskOne(&survey.Confirm{
+			Message: fmt.Sprintf("Are you sure to publish path? (%s)", c.path),
```
Comment (ilgooz) [inline commands/service_publish.go]:
```suggestion
			Message: fmt.Sprintf("Are you sure to publish from path? (%s)", c.path),
```

**Comment by antho1404** [comment]
I would recommend to wait the full feature finished before thinking about ux, for now this features needs a lot of additional work

**Comment by ilgooz** [comment]
> I would recommend to wait the full feature finished before thinking about ux, for now this features needs a lot of additional work

to me this is not for ux. it is to secure personal files against uploading them to ipfs by mistake. for example, even us can give a wrong path to publish cmd like home folders, documents folder etc. and they'll be published to ipfs which we cannot take it back

**Comment by NicolasMahe** [comment]
> > I would recommend to wait the full feature finished before thinking about ux, for now this features needs a lot of additional work
> 
> to me this is not for ux. it is to secure personal files against uploading them to ipfs by mistake. for example, even us can give a wrong path to publish cmd like home folders, documents folder etc. and they'll be published to ipfs which we cannot take it back

I agree with Anthony.

The current implementation of the publish function doesn't integrate the marketplace on ethereum. A lot's of confirmation will be required already..
Let's wait for marketplace integration ;)

**Comment by antho1404** [comment]
this command anyway for now is in hidden so people will not do that mistake if we never document this command

**Comment by ilgooz** [comment]
I'm closing this PR for now

**Comment by ilgooz** [comment]
related to https://github.com/mesg-foundation/core/pull/755#pullrequestreview-199345577

**Pull request closed without merging**